packages/demo-cards/claim-list.gts

```diff
@@ -0,0 +1,50 @@
+import { FieldContainer } from '@cardstack/boxel-ui';
+import {
+  contains,
+  linksToMany,
+  field,
+  Card,
+  Component,
+} from 'https://cardstack.com/base/card-api';
+import StringCard from 'https://cardstack.com/base/string';
+import { Claim } from './claim';
+
+export class ClaimList extends Card {
+  static displayName = 'List of Claims';
+  @field about = contains(StringCard);
+  @field claims = linksToMany(() => Claim);
+  @field title = contains(StringCard, {
+    computeVia: function (this: ClaimList) {
+      return this.about;
```

Comment:
How come `about` and `title` are the same? Should `about` be more something like a description?

Reply:
Resolved https://github.com/cardstack/boxel/pull/418/commits/06083643c95b5989741b62375b552a884b320692 and https://github.com/cardstack/boxel/pull/418/commits/06083643c95b5989741b62375b552a884b320692. I dont know a good name for title so I made it into displayName

Reply:
<img width="1354" alt="Screenshot 2023-06-26 at 20 31 21" src="https://github.com/cardstack/boxel/assets/8165111/cb03bd49-06ab-4d97-937e-0a6ca0629bc4">